modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -23,6 +23,18 @@ definition {
 		AssertElementPresent(locator1 = "FormViewBuilder#SIDEBAR_BASIC_TAB");
 	}
 
+	macro addFieldToObject {
+		Click(locator1 = "FormViewBuilder#OBJECT_FIELD_PLUS");
+
+		Click(
+			key_fieldName = "Add Field to Object",
+			locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
+
+		Click(
+			key_fieldName = "${fieldName}",
+			locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
+	}
+
```

Comment:
Hey @nkkwon , I would suggest you to create a new macro that will validate that addFieldToObject was able add the field. You will just check if the new field was added to the object's sidebar.